doc/rpm-plugin-selinux.8

```diff
@@ -0,0 +1,19 @@
+.TH "RPM-SELINUX" "8" "14 Apr 2016" "Red Hat, Inc."
+.SH NAME
+rpm-plugin-selinux \- SELinux plugin for the RPM Package Manager
+
+.SH Description
+
+The plugin sets SELinux contexts for installed files and executed
+scriptlets. It needs SELinux to be enabled to work but will work in
+both enforcing and permissive mode.
+
+.SH Configuration
+
+There are currently no options for this plugin in particular. See
+.BR rpm-plugins (8)
+on how to control plugins in general.
+
```

Comment:
Hmm, there was a mention of --nocontexts here, which seemed quite appropriate to me. Any particular reason for removing it or did it just get flushed with the generic change?

Reply:
Nah, this is just me being brain dead. Restored.